src/bijectors/truncated.jl

```diff
@@ -0,0 +1,92 @@
+#######################################################
+# Constrained to unconstrained distribution bijectors #
+#######################################################
+struct TruncatedBijector{T} <: Bijector{0}
```

Comment:
Does the parameter here make things type stable?

Reply:
Depends on what the alternative is I suppose, but for example
```julia
struct TruncatedBijector <: Bijector{0}
    lb
    ub
end
```
would at least not be type-stable. But you could of course fix the types to be, say, `Float64`, but that seems unnecessary